src/app-pages/map/components/map-details/MapDetailsContainer.js

```diff
@@ -16,14 +16,22 @@ const MapDetailsContainer = ( props ) => {
     /** @type a2w.models.LocationDetail */
     locationDetailData,
     doUpdateQuery,
+    doLocationDetailFetch,
+    doLocationLevelFetch,
   } = props;
 
   const containerRef = useRef(null);
   const headerRef = useRef(null);
   const display = displayTypes[queryObject.display] ? queryObject.display : defaultMapParams.display;
+  const locationId = queryObject.locationId || "";
+  useEffect(() => {
```

Comment:
Random question: would it make sense to only trigger the fetches if the locationId is not null/""? I know the bundles themselves will abort the fetch if the right URL params can't be populated, but just wondering if it's worth not even trying if there is no location ID to use. Thoughts? Pros/Cons?

Reply:
I made it always trigger the fetch in case there's bad data and the user selects a location without a location id. In practice it should never happen, but in case it does then this will reset the location detail data to an empty object. Otherwise if the user selects a location, then selects another location which doesn't have a location id, the previous location data will not be erased.

Reply:
Gotcha, makes sense.